src/modules/github/components/pull-list-by-repo/pull-list-by-repo.tsx

```diff
@@ -27,39 +31,37 @@ export const PullListByRepo: FC<PullListByRepoProps> = ({
 
   return (
     <div>
-      <div className="flex">
-        <input
-          type="checkbox"
+      <div className={'flex'}>
+        <Checkbox
+          id={`${repo}`}
           checked={isRepoChecked}
-          onChange={() => handleRepoChange(repo)}
+          onCheckedChange={() => handleRepoChange(repo)}
           disabled={!hasChild}
         />
-        <span className="ml-2">{repo}</span>
-        {hasChild && (
-          <svg
-            xmlns="http://www.w3.org/2000/svg"
-            width="10"
-            height="10"
-            viewBox="0 0 100 100"
-            className={'ml-2' + (isRepoOpen ? ' rotate-180' : ' rotate-90')}
-            onClick={handleRepoClick}
-          >
-            <polygon points="50,0 100,100 0,100" fill="black" />
-          </svg>
-        )}
+        <Label htmlFor={`repoList.${repo}`} className="ml-2">
```

Comment:
Wrong. this `htmlFor` value must be the same as `Checkbox` component id above (line 36)